src/game/__tests__/eop.test.js

```diff
@@ -41,27 +42,40 @@ describe('game', () => {
     );
   });
 
-  it('should start with the play phase', () => {
-    expect(players["0"].getState().ctx.phase).toBe('play');
+  it('should start in play stage', () => {
+    expect(players['0'].getState().ctx.activePlayers).toBeFalsy();
   });
 
+  it('should start with no activePlayers', () => {
+    expect(players['0'].getState().ctx.activePlayers).toBeFalsy();
+  })
+
   it('should start with scores set to zero', () => {
     expect(players["0"].getState().G.scores).toStrictEqual([0,0,0]);
   });
 
-  it('should respect the play order', () => {
-    expect(players["0"].getState().G.order[0]).toBe(parseInt(players["0"].getState().ctx.currentPlayer));
+  it('have correct order in ctx', () => {
+    const state = players['0'].getState();
+    expect(state.ctx.playOrder).toStrictEqual(['0', '1', '2']);
+  });
+
+  const startingPlayer = players['0'].getState().ctx.currentPlayer;
```

Comment:
It's strange to define a constant between the tests.

Reply:
So I wasn't really sure how I should do this, I wanted to check that the `currentPlayer` changes to the next player once they've made their move. This just records the player that starts, I think maybe I can access the initial player in G/ ctx though so maybe this would be a bit cleaner.

Reply:
I didn't work out a way to do this without saving who the starting player is somewhere but I've moved it to the top so it's not so out of place